src/content/docs/en/develop-and-build.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Develop and build with Astro
```

Comment:
Do we really need `with Astro` here? The title feels more concise without it, especially thinking this is going to be a BIG heading in the sidebar for translations
```suggestion
title: Develop and build
```